daemon/daemon_unix.go

```diff
@@ -308,11 +308,17 @@ func verifyContainerResources(resources *containertypes.Resources, sysInfo *sysi
 		logrus.Warnf("Your kernel does not support CPU cfs period. Period discarded.")
 		resources.CPUPeriod = 0
 	}
+	if resources.CPUPeriod > 0 && resources.CPUPeriod < 1000 {
+		return warnings, fmt.Errorf("CPU cfs period can not be less than 1ms (i.e. 1000)")
+	}
 	if resources.CPUQuota > 0 && !sysInfo.CPUCfsQuota {
 		warnings = append(warnings, "Your kernel does not support CPU cfs quota. Quota discarded.")
 		logrus.Warnf("Your kernel does not support CPU cfs quota. Quota discarded.")
 		resources.CPUQuota = 0
 	}
+	if resources.CPUQuota > 0 && (resources.CPUQuota < 1000 || resources.CPUQuota > 1000000) {
```

Comment:
Is there a upper bound for `cpu_quota`?  I think there is no upper bound for `cpu_quota` since there could be multiple processors. 
<pre><code>$ docker run -ti --rm --cpu-quota 20000000 busybox
2016/03/18 04:12:54 Error resolving syscall name copy_file_range: could not resolve name to syscall - ignoring syscall.
2016/03/18 04:12:54 Error resolving syscall name mlock2: could not resolve name to syscall - ignoring syscall.
/ #
</code></pre>